api/internal/localizer/localizer.go

```diff
@@ -242,6 +244,48 @@ func (lc *localizer) localizeGenerator(generator *types.GeneratorArgs) (err erro
 	return nil
 }
 
+// localizeHelm localizes helm values files and copies local chart homes.
+func (lc *localizer) localizeHelm(kust *types.Kustomization) error {
```

Comment:
Overall the logic is clear to me. I think it can be more readable to further separate the paths between`HelmInflationGenerator`   and `helmCharts`, especially the logic around `useDefaultChartHome` is a little bit twisted.

something like
```go
//!! no `localizeHelm`

// localizeHelmInflationGenerator localizes the syntax "helmChartInflationGenerator"
func localizeHelmInflationGenerator() {
} 

// localizeHelmCharts localizes the builtin "helmCharts" ...
func localizeHelmCharts() {
} 
```

Reply:
Done. I split the original localize function.